.github/workflows/bump-all-platforms.yml

```
@@ -0,0 +1,219 @@
+              --base "$BASE_BRANCH" \
+              --title "Bump version to \`v${{ steps.version.outputs.core_version }}\` across all platforms" \
+              --body "Unified version bump.
+
+          - **core** (\`nym-vpn-core/Cargo.toml\`): \`${{ steps.version.outputs.core_version }}\`
+          - **tauri app** (\`nym-vpn-app/src-tauri/Cargo.toml\`, \`vpnd_compat.toml\`, \`tauri.conf.json\`): \`${{ steps.version.outputs.core_version }}\` (deb \`${{ steps.version.outputs.deb_version }}\`)
+          - **android** (\`Constants.kt\`): \`VERSION_NAME = \"${{ steps.version.outputs.android_version_name }}\"\`, \`VERSION_CODE = ${{ steps.version.outputs.android_version_code }}\`
+          - **apple** (\`NymVPN.xcodeproj\`): \`MARKETING_VERSION = ${{ steps.version.outputs.semver }}\` (CURRENT_PROJECT_VERSION auto-bumped +1)"
+
+      - name: Dry-run summary
+        if: ${{ inputs.dry_run == true }}
+        run: echo "::warning:: Dry run — no commit or PR created."
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)